### PR TITLE
[Docs]Adding the option of using docker registry

### PR DESCRIPTION
**Pull request by Delawen**
As described in https://camel.apache.org/camel-k/1.9.x/installation/platform/kind.html

### CODE REVIEW

File: docs/modules/ROOT/pages/contributing/local-development.adoc [hunk 1]
```diff
@@ -65,4 +65,20 @@ Camel K integrations are based on https://github.com/apache/camel-k-runtime[Came
...
+Alternatively, if no local registry is available, you can use a docker registry. First, create a secret with your registry username and password:
+
+```
+kubectl -n default create secret docker-registry external-registry-secret --docker-username my-user --docker-password "password"
+```
+
+And then you can run the installation of Camel K operator:
+
+```
+./kamel install --skip-operator-setup -n operator-test --force --runtime-version 1.7.0-SNAPSHOT --registry docker.io --organization my-org-or-username --registry-secret external-registry-secret
```
Comment (oscerd) [inline docs/modules/ROOT/pages/contributing/local-development.adoc:81]: I don't think we should use 1.7.0-SNAPSHOT as runtime version as default in the docs.

**Review by oscerd — CHANGES_REQUESTED** [review]
1 minor finding.

File: docs/modules/ROOT/pages/contributing/local-development.adoc (hunk 1 above)
Comment (Delawen) [reply docs/modules/ROOT/pages/contributing/local-development.adoc:81]: I am using the same version number that is in the same documentation page. Let me replace it with a variable.

**Review by tadayosi — COMMENTED** [review]
Looks nice but I feel it's somewhat a duplicate of this doc:
https://camel.apache.org/camel-k/1.9.x/installation/registry/dockerhub.html

Possibly we can just suggest a link to the installation guide here instead?

**Comment by Delawen** [comment]
Changed by a link to the registry section.

**Review by oscerd — APPROVED** [review]

**Comment by tadayosi** [comment]
@Delawen @oscerd Can we port this to main as well?

**Comment by oscerd** [comment]
Sure.

**Comment by oscerd** [comment]
Done.

**Pull request merged**